cases/scenarios/xstate-main-1/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+cd "${0%/*}"
+
+export SCENARIO_REF=40ec4fab867d150f71fcd15ce248cbc7f95fe87a
```

Comment:
This one should now work with TS 5.4
```suggestion
export SCENARIO_REF=51b844230a26962c6ff351908eb08170567b629f
```